locales/en.json

```diff
@@ -20,6 +20,12 @@
     "internal": "Sorry, we’re unable to process your request. Try again later.",
     "contactService": "Please contact the service you are trying to make a payment to."
   },
+  "paymentLinkError": {
+    "title": "An error occurred:",
+    "invalidReference": "Reference must be 255 characters or fewer. You cannot use any of the following characters < > ; : ` ( ) \" ' = | \",\" ~ [ ]",
+    "invalidAmount": "Enter an amount in pence. For example “2000”.",
```

Comment:
Good spot that it’s actually pence! I think we usually use single quotation marks for things like this — please check.

Reply:
Changed as suggested. (Example of single quotes in design documentation [here](https://design-system.service.gov.uk/components/error-message/label/index.html)).